manifest/README.md

```diff
@@ -0,0 +1,44 @@
+# Manifests
+
+This directory contains manifest files that list YAML files in this repository.
+
+## Subdirectories
+
+### standard
+
+Contains automatically generated manifests from the GEDCOM standards:
+
+#### manifest-5.5.1-en-US.tsv
+
+This table can be used to look up the location of all standard YAML files for GEDCOM 5.5.1.
+
+The table has the following key columns:
+* yaml_path: The path, relative to the root of this repository, of the YAML file.
+
+#### manifest-7.0-en-US.tsv
+
+This table can be used to look up the location of all standard YAML files for the FamilySearch GEDCOM 7.0 standard.
+
+The table has the following key columns:
+* yaml_path: The path, relative to the root of this repository, of the YAML file.
+
+#### manifest-extensions-en-US.tsv
+
+This table can be used to look up the location of the YAML file for a given undocumented extension in a given language.
+
+The table has the following key columns:
+* used_by: The `HEAD`.`SOUR` value of the product using the extension.
+    
+    The special value "`-`" means there are no currently registered `HEAD`.`SOUR` values for this extension.
+* tag: The extension tag.
+* language: The language tag for the YAML file.
+
+The table has the following non-key columns:
+* yaml_path: The path, relative to the root of this repository, of the YAML file.
+
+These files are generated by `../registry_tools/makeTSV.py` and should not be manually updated.
```

Comment:
What does "these files" refer to here? Should it have been up near line 9 instead?